internal/service/alarms/internal/alertmanager/alertmanager.yaml

```diff
@@ -12,7 +12,7 @@ route:
   - receiver: "null"
     matchers:
       # Always firing alert to verify alertmanager is working
-      - Watchdog
+      - alertname="Watchdog"
```

Comment:
Wonder if we should Unmarshal it to Go struct to possible check for any config error validation.